src/main/java/org/generationcp/middleware/service/api/study/VariableDTO.java

```diff
@@ -0,0 +1,604 @@
+package org.generationcp.middleware.service.api.study;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class VariableDTO {
```

Comment:
Hi @abatac - I'm wondering if for Middleware we can return MeasurementVariable and just transform to VariableDTO when we get to BMSAPI? This way we don't have to create yet another variable-related DTO in Middleware.

Reply:
I've actually thought about that before but I decided to return VariableDTO directly from the query instead because if we use MeasurementVariable, then we won't be able to populate some of the fields like synonyms, ontology per variable, etc.